CONDUCT.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "CONDUCT.md"
+output: html_document
+---
+
+---
+title: "Code of Conduct"
+output: html_notebook
+---

```

Comment:
You don't need duplicates of the YAML header. Only one YAML header is allowed.

In fact, this file doesn't need a YAML header at all! :)